ignite/contrib/handlers/neptune_logger.py

```diff
@@ -1,3 +1,4 @@
+# mypy: no-warn-unused-ignores
```

Comment:
@sayantan1410 do you know which objects/variables are unused ? 
My question is why we put this and can't we fix the warning otherwise ?

Reply:
Like instead of ignoring the type errors for the entire file should I do it for like each of the variable or something like that ?

Reply:
Yes, the purpose is to fix the error instead of ignoring it

Reply:
Okay, trying to fix that. Thank you